fdbserver/team/TestBase.actor.cpp

```diff
@@ -0,0 +1,749 @@
+#include <cinttypes>
+#include <deque>
+#include <iomanip>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <tuple>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+#include "fdbclient/CommitTransaction.h"
+#include "fdbclient/FDBTypes.h"
+#include "fdbrpc/fdbrpc.h"
+#include "fdbserver/Knobs.h"
+#include "flow/crc32c.h"
+#include "flow/DeterministicRandom.h"
+#include "flow/UnitTest.h"
+
+#include "flow/actorcompiler.h"
+
+static const int NUM_TEAMS = 12;
+static const int NUM_TLOG_PROCESSES = 2;
+static const int NUM_STORAGE_SERVER_PROCESSES = 2;
+
+namespace Team {
+
+struct TeamID {
+	constexpr static FileIdentifier file_identifier = 345787;
+
+	int32_t teamID;
+
+	std::string toString() const { return std::string(format("%" PRId32, teamID)); }
+	bool operator==(const TeamID& another) const { return this->teamID == another.teamID; }
+
+	template <typename Ar>
+	void serialize(Ar& ar) {
+		serializer(ar, teamID);
+	}
+};
+
+} // namespace Team
+
+namespace std {
+template <>
+struct hash<Team::TeamID> {
+	size_t operator()(const Team::TeamID& teamID) const { return teamID.teamID; }
+};
+} // namespace std
+
+namespace Team {
```

Comment:
We need to define what namespace name to use. `Team` is short, which is good. Maybe all small case, e.g., `team`?